Automattic/wp-calypso
# Marketplace: Update alt & display name on submission success page

#### Proposed Changes
According to comments here, we should change these:
- Show the `display_name` instead of `username` in the `Hello, ` message.
- Remove the `alt` on the `submissionSuccessImage` image.

#### Testing Instructions

1. Go to `/marketplace/submission-success?returning=true`
2. Check showed name.
3. Open the developers console and check for the `alt` on the image (should be empty)

#### Screenshot
| Before | After |
| -- | -- |
| <img width="637" alt="image" src="https://user-images.githubusercontent.com/402286/182918880-5006a279-d8b1-46f7-aa3c-370e4bc86d70.png"> | <img width="637" alt="image" src="https://user-images.githubusercontent.com/402286/182919051-cf36ff63-c713-4155-8d08-f16f7975d8d6.png"> |

#### Pre-merge Checklist

Fixes #66275
Fixes #66273
